code/datums/components/telegraph_ability.dm

```diff
@@ -0,0 +1,39 @@
+/**
+ * Component given to creatures to telegraph their abilities!
+ */
+/datum/component/telegraph_ability
```

Comment:
Should probably be renamed, as we have `/datum/component/basic_mob_attack_telegraph`. Maybe like `/datum/component/basic_mob_ability_telegraph`.